packages/graphql/tests/integration/types/duration.int.test.ts

```diff
@@ -189,7 +189,7 @@ describe("Duration", () => {
 
             const mutation = /* GraphQL */ `
                     mutation ($id: ID!, $duration: Duration) {
-                        ${Movie.operations.update}(where: { id_EQ: $id }, update: { duration_SET: $duration }) {
+                        ${Movie.operations.update}(where: { id_EQ: $id }, update: { duration: { set: $duration } }) {
```

Comment:
same reason as above:
```suggestion
                        ${Movie.operations.update}(where: { id: { equals: "${id}" } }, update: { duration: { set: $duration } }) {
```